src/main/java/org/broadinstitute/hellbender/dev/tools/walkers/bqsr/BaseRecalibrationArgumentCollection.java

```diff
@@ -24,7 +24,11 @@ public BaseRecalibrationArgumentCollection() {}
     protected final RecalibrationArgumentCollection RAC = new RecalibrationArgumentCollection();
 
     @ArgumentCollection
-    public final ReadInputArgumentCollection readArguments = new OptionalReadInputArgumentCollection();
+    public final OptionalReadInputArgumentCollection readArguments = new OptionalReadInputArgumentCollection();
```

Comment:
I thought I had to make it optional because on the worker side it doesn't expect read inputs; but it turns out now that it's fine to leave it as required.